prime-router/src/main/kotlin/cli/tests/CompareData.kt

```diff
@@ -348,19 +348,21 @@ warnings: ${warnings.joinToString()}
 
     /**
      * Compares two input streams, an [actual] and [expected], of the same [schema] and [format].
+     * @param fieldsToIgnore is a list of fields that should be ignored when doing a comparison of field values.
      * @return the result for the comparison, with result.passed true if the comparison was successful
      */
     fun compare(
         expected: InputStream,
         actual: InputStream,
         format: Report.Format?,
         schema: Schema?,
-        result: Result = Result()
+        result: Result = Result(),
+        fieldsToIgnore: List<String>? = null
```

Comment:
Added the ability to exclude some fields from comparison. The `fieldsToIgnore` param is a list of the field names to ignore when doing a comparison.

Reply:
Love this feature.